src/Dockerfile

```diff
@@ -11,10 +11,10 @@ RUN apt-get update && \
     /bin/bash ./build.sh
 
 # Run Phase
-FROM node:8.9.1 as runtime
+FROM node:14.18-bullseye-slim as runtime
 COPY --from=builder /app/gmg-client /app/gmg-client
 COPY --from=builder /app/gmg-server /app/gmg-server
-COPY --from=builder /app/gmg-app/build /app/gmg-server/public
+COPY Dockerfile /
```

Comment:
Did you mean to copy the `Dockerfile` into the image? You also need `--from=builder /app/gmg-app/build /app/gmg-server/public`, as that is the compiled React front end.

Reply:
I did, because then in the image a user can see how it was constructed. 

You don't need that line because `npm run publish` in the build script effectively runs `"rimraf ../gmg-server/public/app && npm run build && mv ./build ../gmg-server/public/app` and once that happens, the `COPY` statement fails because the source has been moved and not copied.